src/components/index/Banner.tsx

```diff
@@ -97,9 +97,9 @@ const Banner = () => (
                 <header role="banner" className="banner pattern">
                     <div className="banner__text-box">
                          <h1>
-                            Dev Environment as Code <span>for modern DevOps teams.</span>
+                            Dev Environments as Code <span>for modern DevOps teams.</span>
                         </h1>
-                        <p>Describe your dev environment as code and get a fully prebuilt, ready-to-code development environment for any GitLab, GitHub, and Bitbucket project.</p>
+                        <p>Describe your dev environment as code and get fully prebuilt, ready-to-code development environments for any GitLab, GitHub, and Bitbucket project.</p>
```

Comment:
I personally prefer the old sentence (less confusion around "I don't want to get fully prebuilt myself")

```suggestion
                        <p>Describe your dev environment as code and get a fully prebuilt, ready-to-code development environment for any GitLab, GitHub, and Bitbucket project.</p>
```